RELEASE.md

```diff
@@ -20,29 +20,35 @@ allows intermediate publishing from the main branch using version X.Y.Z.dev\<N\>
 ## Cutting the release
 Creating the release involves
 
+1. Edit the `release-notes.md` to list major/minor changes
 1. Creating a release branch and tag and updating the main branch versions.
 1. Creating a github release from the release branch and tag.
 1. Building and publishing pypi library wheels and docker registry image.
```

Comment:
Is this missing the step for creating a new dev branch that increments the release number ?

Also should we use Create a ..., Build and publish... similar to first bullet.

Reply:
i think that is covered in 2 `...and updating the main branch versions.`